packages/base/Badge/src/Badge/Badge.tsx

```diff
@@ -49,25 +45,37 @@ export const Badge = forwardRef<HTMLDivElement, Props>(function Badge(
   },
   ref
 ) {
-  const classes = useStyles()
-
   const hasChildren = Children.count(children) > 0
 
   return (
     <MuiBadge
       ref={ref}
-      style={style}
       data-testid={testId}
       badgeContent={content}
       max={max || thresholds[size]}
-      className={className}
       showZero
-      classes={{
-        badge: cx(classes.root, classes[variant], classes[size], {
-          [classes.static]: !hasChildren,
-        }),
+      slotProps={{
+        root: {
+          className: twMerge(
+            `inline-flex flex-shrink-0 content-middle flex-nowrap justify-normal 
+            text-[10px] font-semibold leading-3 align-middle text-graphite-700 
+            relative top-0 right-0`,
+            hasChildren ? 'relative' : 'static',
+            className
+          ),
+          style: style,
+        },
+        badge: {
+          className: twMerge(
+            `border-solid items-center content-center inline-flex flex-nowrap 
+            justify-center z-[1] border px-[1px] border-gray-400 static`,
```

Comment:
```suggestion
            justify-center z-[1] border px-[1px]`,
```
we already set border color in classbyVariant. Also if we move the `static` to the `hasChildren` condition, we don't need to use `twMerge` here as it is the only thing we merge `absolute -> static`
```suggestion
            justify-center z-[1] border px-[1px] border-gray-400 static`,
```

Reply:
ok also `px-[1px]` would need to be moved to `small` and `medium` size. But I would try not to use twMerge for these if it is not necessary